src/main/java/org/stuartgunter/dropwizard/cassandra/CassandraFactory.java

```diff
@@ -85,6 +85,11 @@
  *         <td>Sets the compression to use for the transport. Must a value in the {@link ProtocolOptions.Compression compression enum}.</td>
  *     </tr>
  *     <tr>
+ *         <td>maxSchemaAgreementWait</td>
+ *         <td>No default.</td>
+ *         <td>Sets the maximum time in seconds to wait for schema agreement before returning from a DDL query.</td>
```

Comment:
Config values for durations shouldn't be constrained to a particular unit of time. Please could you change this to use the `Duration` type provided by Dropwizard for this purpose. This type allows a much more expressive way to configure systems, using whatever time period is most appropriate (e.g. why write 120 seconds when 2 minutes will do?)